--- origsrc/wxstedit/Makefile.in	2007-03-06 17:57:59.000000000 -0600
+++ src/wxstedit/Makefile.in	2008-05-21 22:22:16.734375000 -0500
@@ -68,7 +68,7 @@
 	stedit_lib_stestyls.o
 STEDIT_LIB_HEADERS =  \
 	include/wx/stedit/pairarr.h \
-	include/wx/stedit/setup0.h \
+	include/wx/stedit/setup.h \
 	include/wx/stedit/stedefs.h \
 	include/wx/stedit/stedit.h \
 	include/wx/stedit/stedlgs.h \
@@ -106,7 +106,7 @@
 	stedit_dll_stestyls.o
 STEDIT_DLL_HEADERS =  \
 	include/wx/stedit/pairarr.h \
-	include/wx/stedit/setup0.h \
+	include/wx/stedit/setup.h \
 	include/wx/stedit/stedefs.h \
 	include/wx/stedit/stedit.h \
 	include/wx/stedit/stedlgs.h \
