--- origsrc/wxstedit/Makefile.in	2007-03-06 17:57:59.000000000 -0600
+++ src/wxstedit/Makefile.in	2008-05-21 22:22:16.734375000 -0500
@@ -68,7 +68,7 @@
 	stedit_lib_stestyls.o
 STEDIT_LIB_HEADERS =  \
 	include/wx/stedit/pairarr.h \
-	include/wx/stedit/setup0.h \
+	include/wx/stedit/setup.h \
 	include/wx/stedit/stedefs.h \
 	include/wx/stedit/stedit.h \
 	include/wx/stedit/stedlgs.h \
@@ -106,7 +106,7 @@
 	stedit_dll_stestyls.o
 STEDIT_DLL_HEADERS =  \
 	include/wx/stedit/pairarr.h \
-	include/wx/stedit/setup0.h \
+	include/wx/stedit/setup.h \
 	include/wx/stedit/stedefs.h \
 	include/wx/stedit/stedit.h \
 	include/wx/stedit/stedlgs.h \
--- origsrc/wxstedit/src/stenoteb.cpp	2007-06-19 16:23:26.000000000 -0500
+++ src/wxstedit/src/stenoteb.cpp	2008-05-21 22:39:21.937500000 -0500
@@ -757,7 +757,11 @@
         wxFileDialog fileDialog( this, _("Open file(s) into new notebook page"),
                                  GetOptions().GetDefaultFilePath(),
                                  wxEmptyString, extensions,
+#if wxCHECK_VERSION(2, 7, 0)
                                  wxFD_OPEN | wxFD_MULTIPLE );
+#else
+                                 wxOPEN | wxMULTIPLE );
+#endif
 
         if (fileDialog.ShowModal() == wxID_OK)
             fileDialog.GetPaths(filePaths);
